src/DateUtils.js

```diff
@@ -15,9 +15,36 @@ limitations under the License.
 */
 
 'use strict';
+import _t from 'counterpart';
 
-var days = ["Sun", "Mon", "Tue", "Wed", "Thu", "Fri", "Sat"];
-var months = ["Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"];
+function getDaysArray() {
+	var days = [];
+	days.push(_t('Sun'));
```

Comment:
failing that, this definitely needs a comment to explain why the weirdness is happening :)